src/config/datasets.yml

```diff
@@ -300,25 +300,81 @@ collections:
   landsat-8-c2-l2:
     isHidden: true
 
+  modis-09A1-061:
```

Comment:
@pholleway I added this initial config that matches a hero image to a collection and adds a category (we can change these if they're not right). This is also where references to the notebooks will go when they're ready.